scripts/build.sh

```diff
@@ -157,9 +157,6 @@ tools_build_muon() {
 
     CC="${CC}" ninja="${SAMU}" stage1/muon setup        \
         -Dprefix="${BUILDDIR}/build-tools"              \
-        -Dlibcurl=enabled                               \
-        -Dlibarchive=enabled                            \
-        -Dlibpkgconf=enabled                            \
```

Comment:
This will not work. The only reason why it is passing the CI build is because the `muon` used is prebuild which has support for fetching libnvme.

Reply:
That is surprising. It builds without these package. `muon` uses `git` directly, that's why it still works...